v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -0,0 +1,93 @@
+package com.google.cloud.teleport.v2.elasticsearch.options;
```

Comment:
can you add license header similar to [existing format](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/master/v2/pom.xml).  Applies to all other files as well.